README.md

```diff
@@ -39,15 +39,15 @@ The iOS SDK is available through [**CocoaPods**](https://cocoapods.org), [**Cart
 To install the SDK, simply add the following line to your Podfile:
 
 ```xcode
-pod 'Rudder', '1.1.5'
+pod 'Rudder', '1.2.5'
```

Comment:
Version should be 1.1.6

Reply:
Changed version to 1.2.0 as we discussed